core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -5226,7 +5234,24 @@ private static void computeVersion(ServletContext context) {
     private static final Logger LOGGER = Logger.getLogger(Jenkins.class.getName());
 
     public static final PermissionGroup PERMISSIONS = Permission.HUDSON_PERMISSIONS;
+    /**
+     * Grants ability to configure any and all aspects of the Jenkins instance
+     */
     public static final Permission ADMINISTER = Permission.HUDSON_ADMINISTER;
+
+    /**
+     * Allows non-privilege escalating configuration permission for a Jenkins instance.  Actions which could result
+     * in a privilege  escalation (such as RUN_SCRIPTS) require explicit ADMINISTER permission.
+     *
+     * As an experimental feature, making the manage permission able to be disabled by default (keep as ADMINISTER), can
+     * be enabled with "jenkins.security.ManagePermission" system property.
+     */
+    public static final Permission MANAGE = new Permission(PERMISSIONS, "Manage",
```

Comment:
where we not trying to mark this as `@Beta` for the moment?

Reply:
The interface for this permission is not expected to change. Why should we consider it Beta?

Reply:
This is still intended to be delivered as an experimental feature, right?

If so, makes sense to require a minor hurdle to use it. Strictly speaking, all the `#getRequiredPermission()` and such should be annotated, but this one's probably more straightforward.